docs/contributing/developing.rst

```diff
@@ -27,12 +27,6 @@ Clone a copy of `the Wagtail codebase <https://github.com/wagtail/wagtail>`_:
     $ git clone https://github.com/wagtail/wagtail.git
     $ cd wagtail
 
-With your preferred virtualenv activated, install the Wagtail package in development mode with the included testing and documentation dependencies:
```

Comment:
Don't we still want this section here?

Reply:
I would think so — thought it does make me think that perhaps where we mention "virtualenv" we should link to something like [the Python virtual environment docs](https://docs.python.org/3/tutorial/venv.html). It might also be a good idea to say "virtual environment" (like we do in the [README](https://github.com/wagtail/wagtail#getting-started) and the [Getting Started documentation](http://docs.wagtail.io/en/v2.5.1/getting_started/index.html#quick-install)). That way someone's not left wondering what that means.

Reply:
+1 on both points

Reply:
I moved it back to be under the same section